appveyor.sh

```diff
@@ -62,6 +62,18 @@ for proj in druntime phobos; do
     fi
 done
 
+# download and install Visual D for VS 2015
+if [ ! -f "VisualD-v0.45.1-rc2.exe" ]; then
+    appveyor DownloadFile "https://github.com/dlang/visuald/releases/download/v0.45.1-rc2/VisualD-v0.45.1-rc2.exe" -FileName 'VisualD-v0.45.1-rc2.exe'
+fi
+.\VisualD-v0.45.1-rc2.exe /S
```

Comment:
How about making the release version a variable? You use it four times here and a fifth for the cache.

Reply:
Agreed. Setting a variable now, I suspect it cannot be used in the cache settings even if defined in appveyor.yml